### ISSUE TITLE
Uncaught Error: Invalid LatLng object: (NaN, NaN) when zooming out

### ISSUE THREAD

**Issue by jywarren** [publiclab/leaflet-blurred-location#93]
I see this error at https://publiclab.org/locations/form when i start zooming the map. @mridulnagpal - any idea why?

```
VM4334:532 Uncaught Error: Invalid LatLng object: (NaN, NaN)
    at M (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:2603)
    at C (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:2823)
    at T.extend (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:20013)
    at T (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:2493)
    at z (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:2546)
    at e._boundsToLatLngs (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:122265)
    at e.initialize (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:122131)
    at new e (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:14439)
    at Object.t.rectangle (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:532:136921)
    at drawCenterRectangle (eval at <anonymous> (jquery.js?body=1:339), <anonymous>:751:19)
```

**Comment by mridulnagpal** [comment]
Made a PR most probably this was because it was sending NaN values in bounds to call rectangle. Added some sort of a filter to avoid that in PR #101

**Comment by jywarren** [comment]
solved in #101!
